csp-mixin/signals/gcpceOverview.libsonnet

```
@@ -0,0 +1,300 @@
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableSentPackets: {
+        name: 'tableSentPackets',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_sent_packets_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableReceivedPackets: {
+        name: 'tableReceivedPackets',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_received_packets_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableSentBytes: {
+        name: 'tableSentBytes',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_sent_bytes_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableReceivedBytes: {
+        name: 'tableReceivedBytes',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_received_bytes_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableReadBytes: {
+        name: 'tableReadBytes',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_disk_read_bytes_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableWriteBytes: {
+        name: 'tableWriteBytes',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_disk_write_bytes_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      // end table
+
+      memoryUtilization: {
+        name: 'Memory Utilization',
+        description: 'Memory Utilization',
+        type: 'raw',
+        unit: 'bytes',
+        sources: {
+          stackdriver: {
+            expr: 'sum(stackdriver_gce_instance_compute_googleapis_com_instance_memory_balloon_ram_size{instance_name=~"$instance_name"} or on() vector(0)) + sum(stackdriver_gce_instance_compute_googleapis_com_guest_memory_bytes_used{instance_name=~"$instance_name"} OR on() vector(0))',
```

Don't understand why you implemented the query in this way. I think if you move this signal and the memoryUsed signal to **gce.libsonnet**, you will have the filter by **instance_name** plus the other filters included and then the query can be changed to something like this:

```suggestion
           expr: 'sum(stackdriver_gce_instance_compute_googleapis_com_instance_memory_balloon_ram_size{%(queriesSelector)s}) + sum(stackdriver_gce_instance_compute_googleapis_com_guest_memory_bytes_used{%(queriesSelector)s})',
```